desc/objectives/_omnigenity.py

```diff
@@ -549,6 +549,13 @@ class Omnigenity(_Objective):
         target_default="``target=0``.", bounds_default="``target=0``."
     )
 
+    _static_attrs = _Objective._static_attrs + [
```

Comment:
`_helicity` should be static, I think, for the QS objectives in this fiile right? tuple of ints?